src/update_lmod_system_cache_files.in

```diff
@@ -334,7 +334,7 @@ set_defaults
 check_parameters
 
 if [ -z "$LMOD_REVERSEMAPT_DIR" ]; then
-  LMOD_REVERSEMAPT_DIR="$LMOD_CACHE_DIR/../reverseMapD"
+  LMOD_REVERSEMAPT_DIR="$LMOD_CACHE_DIR/reverseMapD"
```

Comment:
You get a handle for changing this via `$LMOD_REVERSEMAPT_DIR`, why change the default if you don't like it?